carto/import_api.py

```diff
@@ -131,15 +131,15 @@ def get(self, id=None, ids_only=False):
         :return: An import job, a list of import job IDs or a list of import jobs
         """
         if id is not None:
-            resp = self.client.send("%s/%s" % (self.client.api_url, id))
+            resp = self.client.send("%s/%s" % (self.api_url, id))
             response_data = self.client.get_response_data(resp, True)
             return ImportJob(self.client, **response_data)
         else:
             imports = []
 
-            resp = self.client.send(self.client.api_url)
```

Comment:
why are you removing client? ImportManager doesn't have an api_key attribute, does it?